torch/_inductor/kernel/templated_attention.py

```diff
@@ -0,0 +1,162 @@
+""" Triton Implementation of the Templated SDPA Kernel"""
+import logging
+
+import torch
+from ..select_algorithm import TritonTemplate
+
+log = logging.getLogger(__name__)
+aten = torch.ops.aten
+
+
+def sdpa_grid(batch_size, num_heads, num_queries, d_model, meta):
```

Comment:
This might just be bad comment, since we are parallelizing over quries, it should really be

"we launch batch_size * num_heads * ceil(n_queries/BLOCK_M) programs"

Reply:
let me make some updates to this comment